frontend/packages/ceph-storage-plugin/src/constants/queries.ts

```diff
@@ -49,17 +52,67 @@ export const CAPACITY_USAGE_QUERIES = {
+  [StorageDashboardQuery.PODS_BY_USED]:
+    'sum((avg_over_time(kubelet_volume_stats_used_bytes[1h]) * on (namespace,persistentvolumeclaim) group_right() kube_pod_spec_volumes_persistentvolumeclaims_info) * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)|(ceph.rook.io/block)"})) by (pod,namespace)',
+  [StorageDashboardQuery.PODS_TOTAL_USED]:
+    'sum((avg_over_time(kubelet_volume_stats_used_bytes[1h]) * on (namespace,persistentvolumeclaim) group_right() kube_pod_spec_volumes_persistentvolumeclaims_info) * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)|(ceph.rook.io/block)"})) by (pod,namespace)',
+  [StorageDashboardQuery.CEPH_CAPACITY_TOTAL]: 'ceph_cluster_total_bytes',
+  [StorageDashboardQuery.CEPH_CAPACITY_USED]: 'ceph_cluster_total_used_bytes',
+};
+
+export const breakdownQueryMap = {
+  [PROJECTS]: {
+    model: ProjectModel,
+    metric: 'namespace',
+    queries: {
+      [StorageDashboardQuery.PROJECTS_BY_USED]: `(sort_desc(topk(5,(${
+        CAPACITY_BREAKDOWN_QUERIES[StorageDashboardQuery.PROJECTS_BY_USED]
+      }))))`,
+      [StorageDashboardQuery.PROJECTS_TOTAL_USED]:
+        CAPACITY_BREAKDOWN_QUERIES[StorageDashboardQuery.PROJECTS_TOTAL_USED],
+      [StorageDashboardQuery.CEPH_CAPACITY_TOTAL]:
+        CAPACITY_BREAKDOWN_QUERIES[StorageDashboardQuery.CEPH_CAPACITY_TOTAL],
+      [StorageDashboardQuery.CEPH_CAPACITY_USED]:
+        CAPACITY_BREAKDOWN_QUERIES[StorageDashboardQuery.CEPH_CAPACITY_USED],
+    },
+  },
+  [STORAGE_CLASSES]: {
+    model: StorageClassModel,
+    metric: 'storageclass',
+    queries: {
+      [StorageDashboardQuery.STORAGE_CLASSES_BY_USED]: `(sort_desc(topk(5,(${
+        CAPACITY_BREAKDOWN_QUERIES[StorageDashboardQuery.STORAGE_CLASSES_BY_USED]
+      }))))`,
+      [StorageDashboardQuery.STORAGE_CLASSES_TOTAL_USED]:
+        CAPACITY_BREAKDOWN_QUERIES[StorageDashboardQuery.STORAGE_CLASSES_TOTAL_USED],
+      [StorageDashboardQuery.CEPH_CAPACITY_TOTAL]:
+        CAPACITY_BREAKDOWN_QUERIES[StorageDashboardQuery.CEPH_CAPACITY_TOTAL],
+      [StorageDashboardQuery.CEPH_CAPACITY_USED]:
+        CAPACITY_BREAKDOWN_QUERIES[StorageDashboardQuery.CEPH_CAPACITY_USED],
+    },
+  },
+  [PODS]: {
+    model: PodModel,
+    metric: 'pod',
+    queries: {
+      [StorageDashboardQuery.PODS_BY_USED]: `(sort_desc(topk(5, (${
```

Comment:
```suggestion
      [StorageDashboardQuery.PODS_BY_USED]: `(sort_desc(topk(25, (${
```
Cross verify this data with the data from K8s. To verify that these Pods actually exists. 
https://github.com/openshift/console/blob/master/frontend/public/components/dashboard/dashboards-page/overview-dashboard/top-consumers-card.tsx#L138

Reply:
@rawagner I think it's necessary here too. What do you think?

Reply:
Yes, i agree. This is the same case as with top consumers in utilization card. The resource can be in prometheus still but its already gone in openshift (project/pod/... was deleted) and you dont want to link to 404